### ISSUE TITLE
Incompatibility with message editors 

### ISSUE THREAD

**Issue by pintoso** [LOOHP/InteractiveChat#25]
since I started using InteractiveChat all messages that I try to remove using [Message Editor](https://www.spigotmc.org/resources/message-editor.82154/) are not removed/hidden

after contacting the Message Editor dev he told me:

>Hey, so I looked into this and it looks like this issue is caused by Interactive Chat and not Message Editor. The reason for this issue is that Interactive Chat processes all chat packets, even those that will normally not be even sent to the player. I suggest redirecting this to the Interactive Chat's repository issue tracker. You can also notify the author that the easiest fix I can think of is not processing cancelled packets and moving from the monitor to the highest priority because you are not supposed to edit packets in the monitor priority anyway.

**setup:**
- 1.16.x server
- InteractiveChat latest version + dependencies 
- Message-Editor latest version

**how to reproduce:**
1. start the server
2. open message-editor config.yml
3. add

```
- ==: MessageEdit
  message-before-pattern: \{"translate":"commands\.gamemode\.success\.self"\,"with":\[\{"translate":"gameMode\.creative"\}\]\}
  message-before-place: SYSTEM_CHAT
  message-after: ''
  message-after-place: SYSTEM_CHAT
```
4. change to creative mode

the message saying that you have switched to creative mode will appear, even after it is removed(that is the problem)

**Comment by LOOHP** [comment]
I've just released a new dev build, 3.2.9.5, feel free to give it a try.

**Comment by pintoso** [comment]
it's working, thank you ^^
